### DISCUSSION TITLE
Feature request: Show zoom level in toolbar

### DISCUSSION THREAD

**Discussion by proideo** [sumatrapdfreader/sumatrapdf#4453] [Ideas]
Hi. I think it would be interesting to check the current zoom level in the toolbar.

I have looked into the new toolbar customization feature but have not found a way to add it.

Thanks.

**Comment by kjk** [comment]
In latest [pre-release](https://www.sumatrapdfreader.org/prerelease) you can press `i` to see current page number and zoom level.

In earlier builds it's `I`.

**Reply by proideo** [reply]
But it disappears when I switch tabs. Sometimes I have to compare documents at the same zoom level and that is something I miss in Sumatra.

**Reply by GitHubRulesOK** [reply]
you can set a key to a fixed zoom or press `i` to see if tabs differ or set icon to show zoom (see i in second image)
![image](https://github.com/user-attachments/assets/09d7ed59-b661-409d-a006-73cad1df999e)
![image](https://github.com/user-attachments/assets/b2d97af7-a618-4542-aff4-b7c4c405033b)

**Reply by kjk** [reply]
That's oversight, I've opened https://github.com/sumatrapdfreader/sumatrapdf/issues/4454 to track this

**Reply by luyuduan** [reply]
Hello, I also found that when clicking on the bookmark of the same pdf, there is also this situation 

V3.6.16710 64-bit Pre-release

**Comment by LucaSorvillo** [comment]
Hello, is there any news on this? At the moment i can only see the page number (v3.5.2)
![image](https://github.com/user-attachments/assets/e4f15143-9173-4019-8894-34a73b4f82a8)

**Reply by GitHubRulesOK** [reply]
@LucaSorvillo
 As explained above it can only be seen in the Pre-Release 
![image](https://github.com/user-attachments/assets/c9cee5af-aa7d-4c5c-b08b-8911708378c7)

features cannot be added back to older versions like 3.5.2

**Comment by giwul70** [comment]
+1
I use the Pre-Release version and know about the 'i'
However, I share above suggestion: it would be great if the zoom level (100%, 130%, etc) would be shown in the toolbar.
Please, don't get me wrong .. no offence meant: SumatraPDF is a great tool!

FWIW (not sure if it is all correct) - in Advanced Settings I changed:
DefaultZoom = 100 (initially I changed it in 'Actual Size' - later I noticed 'Actual Size' was changed into 100
ZoomLevels = 
ZoomIncrement = 10
Zoomsteps are now 110,121,133,146,161,177, 195 etc.
(Zoomlevels = 10, changes from 100% to Fit to page as 1st step, and then to 6400% as second step)
